app/views/challenges/_follow_challenge_modal.html.erb

```diff
@@ -6,28 +6,32 @@
 >
   <div class="usa-modal__content maxw-mobile-lg">
     <div class="usa-modal__main">
-      <h2 class="usa-modal__heading font-sans-lg margin-bottom-2" id="follow-challenge-modal-heading">
-        Follow challenge as guest
+      <% if Rails.configuration.gov_delivery_topic_subscribe_url && @challenge.gov_delivery_topic %>
```

Comment:
you can test this by adding this to your ENV (needs a server restart):
```
export GOV_DELIVERY_TOPIC_SUBSCRIBE_URL="https://public.govdelivery.com/accounts/USGSATTS/subscriber/new?topic_id="
```
and also add a `gov_delivery_topic` to a challenge in the rails console
```
challenge.update(gov_delivery_topic: "ABC-123")
```